std/file.d

```diff
@@ -2887,6 +2887,7 @@ private void copyImpl(const(char)[] f, const(char)[] t, const(FSChar)* fromz, co
 
 unittest
 {
+    import std.algorithm, std.file; // issue 14817
```

Comment:
I can't believe you have to import std.file inside std.file. That seems... wrong.